app-load-params-db.json

```diff
@@ -327,6 +327,18 @@
     "curve": ["ed25519"],
     "path": ["44'/1003'", "44'/354'"]
   },
+  "OpenPGP": {
+    "appFlags": {"nanos": "0x240", "nanos2": "0x240", "nanox": "0x240", "stax": "0x240"},
+    "appName": "OpenPGP",
+    "curve": ["secp256k1"],
+    "path": ["2152157255'"]
+  },
+  "OpenPGP.XL": {
+    "appFlags": {"nanos": "0x240"},
+    "appName": "OpenPGP.XL",
```

Comment:
can we use a ` ` instead of a `.` ?

Reply:
Just to keep existing names, avoiding more complications.